.gitignore

```diff
@@ -95,3 +95,7 @@ sitemap.xml
 /test-results/
 /playwright-report/
 /playwright/.cache/
+
+
+queries/clients/types
```

Comment:
It does not work. I still see the types file on the commits.

![image](https://github.com/kodadot/nft-gallery/assets/31397967/94935ea7-cb48-442d-9e4b-f96b048c119a)